app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/menu/DrawerMenu.kt

```diff
@@ -25,7 +25,7 @@ class DrawerMenu @Inject constructor(
     private lateinit var currentNavigationItem: DrawerNavigationItem
 
     fun setup(
-            toolbar: Toolbar,
+            toolbar: Toolbar? = null,
```

Comment:
If a toolbar is optional, Please move to after navigationView like this(You can change this by change signature refactoring) 🔧 .Because parameter can be omitted

```
             navigationView: NavigationView,
             toolbar: Toolbar? = null,
              actionBarDrawerSync: Boolean = false
```